test/AnchorRepository.test.js

```diff
@@ -90,34 +92,24 @@ contract("AnchorRepository", function (accounts) {
 
         it("should only allow fully filled PreAnchors to be recorded", async function () {
             const {anchorId, signingRoot, centrifugeId, precommitSignature, expirationBlock, anchorRepository, callOptions} = await getBasicTestNeeds(accounts);
-
-            await shouldRevert(anchorRepository.preCommit("", signingRoot, centrifugeId, precommitSignature, expirationBlock, callOptions));
-            await shouldRevert(anchorRepository.preCommit(null, signingRoot, centrifugeId, precommitSignature, expirationBlock, callOptions));
-            await shouldRevert(anchorRepository.preCommit(anchorId, "", centrifugeId, precommitSignature, expirationBlock, callOptions));
-            await shouldRevert(anchorRepository.preCommit(anchorId, null, centrifugeId, precommitSignature, expirationBlock, callOptions))
-            await shouldRevert(anchorRepository.preCommit(anchorId, signingRoot, "", precommitSignature, expirationBlock, callOptions));
-            await shouldRevert(anchorRepository.preCommit(anchorId, signingRoot, null, precommitSignature, expirationBlock, callOptions));
-            await shouldRevert(anchorRepository.preCommit(anchorId, signingRoot, centrifugeId, "", expirationBlock, callOptions));
-            await shouldRevert(anchorRepository.preCommit(anchorId, signingRoot, centrifugeId, null, expirationBlock, callOptions));
-            await shouldRevert(anchorRepository.preCommit(anchorId, signingRoot, centrifugeId, precommitSignature, "", callOptions));
-            await shouldRevert(anchorRepository.preCommit(anchorId, signingRoot, centrifugeId, precommitSignature, null, callOptions));
+            ;
+            await shouldRevert(anchorRepository.preCommit("0x0", signingRoot, centrifugeId, precommitSignature, expirationBlock, callOptions));
+            await shouldRevert(anchorRepository.preCommit(anchorId, "0x0", centrifugeId, precommitSignature, expirationBlock, callOptions));
+            await shouldRevert(anchorRepository.preCommit(anchorId, signingRoot, "0x0", precommitSignature, expirationBlock, callOptions));
+            await shouldRevert(anchorRepository.preCommit(anchorId, signingRoot, centrifugeId, "0x0", expirationBlock, callOptions));
+            await shouldRevert(anchorRepository.preCommit(anchorId, signingRoot, centrifugeId, precommitSignature, "0x0", callOptions));
```

Comment:
Were those extra transactions redundant or why did you remove them?

Reply:
Yea, it does param type check and it does not allow you to send empty or null.